src/Xamarin.Android.Build.Tasks/Tests/Xamarin.Android.Build.Tests/BuildTest.cs

```diff
@@ -806,6 +806,18 @@ public void IfAndroidJarDoesNotExistThrowXA5207 ([Values(true, false)] bool buil
 			Directory.Delete (AndroidSdkDirectory, recursive: true);
 		}
 
+		[Test]
+		public void InvalidTargetPlatformVersion ([Values ("net9.0-android33", "net9.0-android33.0")] string targetFramework)
+		{
+			var project = new XamarinAndroidApplicationProject {
+				TargetFramework = targetFramework,
+			};
+			using var builder = CreateApkBuilder ();
+			builder.ThrowOnBuildFailure = false;
+			Assert.IsFalse (builder.Build (project), "build should fail");
+			Assert.IsTrue (builder.LastBuildOutput.ContainsText ("error XA1038:"), "XA1038 should have been raised.");
```

Comment:
The unit test doesn't assert against the error message text, hence [the previous question](https://github.com/xamarin/xamarin-android/pull/8569/files#r1433292062).